web-common/src/components/button-toggle-group/__stories__/ButtonToggleGroup.stories.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+  import { Meta, Story } from "@storybook/addon-svelte-csf";
+
+  import { ButtonToggleGroup, SubButton } from "../index";
+  import Delta from "../../icons/Delta.svelte";
+  import PieChart from "../../icons/PieChart.svelte";
+
+  const pieTooltips = {
+    selected: "Hide percent of total",
+    unselected: "Show percent of total",
+    disabled: "To show percent of total, show top values by a summable metric",
+  };
+
+  const deltaTooltips = {
+    selected: "Hide percent change",
+    unselected: "Show percent change",
+    disabled: "To show percent change, select a comparison period abovec",
+  };
+</script>
+
+<Meta title="Button toggle group stories" />
+
+<Story name="Button toggle group variations (tables)">
```

Comment:
Should one (or all) of these stories show storybook actions to show the on:selected event working?

Reply:
Great idea, added storybook action to show the events working (and not working when a button is disabled)

Reply:
great idea, added actions